packages/manager/src/features/CloudPulse/CloudPulseTabs.tsx

```diff
@@ -0,0 +1,55 @@
+import { styled } from '@mui/material/styles';
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+
+import { SuspenseLoader } from 'src/components/SuspenseLoader';
+import { SafeTabPanel } from 'src/components/Tabs/SafeTabPanel';
+import { TabLinkList } from 'src/components/Tabs/TabLinkList';
+import { TabPanels } from 'src/components/Tabs/TabPanels';
+import { Tabs } from 'src/components/Tabs/Tabs';
+
+import { DashBoardLanding } from './Dashboard/DashboardLanding';
+type Props = RouteComponentProps<{}>;
+
+export const CloudPulseTabs = React.memo((props: Props) => {
+  const tabs = [
+    {
+      routeName: `${props.match.url}/dashboards`,
+      title: 'Dashboards',
+    },
+  ];
+
+  const matches = (p: string) => {
+    return !Boolean(props.location.pathname.indexOf(p));
+  };
```

Comment:
This is the convention I found in several other places in the app:

```suggestion
  const matches = (p: string) => {
    return Boolean(matchPath(p, { path: props.location.pathname }));
  };
```